### ISSUE TITLE
stride issue

### ISSUE THREAD

**Issue by hamnaz** [brichard19/BitCrack#255]
hi
an issue inside stride found
current ver stride work like ( explain in dec)
setting stride
--stride 5F5E100 (dec 100000000)
check key 5F5E101(dec 100000001 ( 129SJkPvcj4oGF2M5DTtNL97JgSbxQYZSG --stride 5F5E100 ), result OK
next check apear  BEBC201 (dec 20000001) (1LejsLkBwCiitUXSD4syGNRp8BnK8hHxGQ --stride 5F5E100) result ok
its working like 
1 start key+(stride-key-1) (stride-key-1)(stridekey-1)
in dec values
1+100000001+200000001+300000001

it should be
1+(stride:100000000)+1(from previous total 100000002 + (stride: 100000000+1) total check point key 200000003
in short 1+100000000+1+100000000+1
check point is 1 and stride is 100000000
could dev or fork puller could make it correct ????
